app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,82 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if !Helpers.logged_in?(session)
+      redirect "/login"
+    else
+      @user = Helpers.current_user(session)
+      @tweets = Tweet.all
+      erb :'/tweets/index'
+    end
+  end
+
+  get '/tweets/new' do
+    if Helpers.logged_in?(session)
+      erb :'/tweets/new'
+    else
+      redirect "/login"
+    end
+  end
+
+  post '/tweets' do
+    @tweet = Tweet.new(content: params[:content])
+    @tweet.user_id = session[:user_id]
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```